commons/zenoh-config/src/lib.rs

```diff
@@ -70,6 +70,24 @@ impl Zeroize for SecretString {
 
 pub type SecretValue = Secret<SecretString>;
 
+#[derive(Debug, Deserialize, Serialize, Clone)]
+#[serde(rename_all = "lowercase")]
+pub enum DownsamplingStrategy {
+    Ratelimit,
+}
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct DownsamplingItemConf {
```

Comment:
See my previous comments on configuration. It may impact on how `DownsamplingItemConf` is defined.